src/app/App.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Since React v17 it's possible not to import React from "react". So let's remove this.